### ISSUE TITLE
Bug ngrx/prefix-selectors-with-select rule does not accept variables with numbers in their name

### ISSUE THREAD

**Issue by alorle** [timdeschryver/eslint-plugin-ngrx#261]
## Behavior

`ngrx/prefix-selectors-with-select` rule does not accept variables with numbers in their name.

## Code snippet

The following code
```ts
export const selectI18nFeature = createFeatureSelector<State>(i18nFeatureKey);
```
throws:
```
warning  The selector should start with "select"  ngrx/prefix-selectors-with-select
```

## I would be willing to submit a PR to fix this issue :heart:

[x] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No
